### PR TITLE
Community

### PR DESCRIPTION
**Pull request by qedrakmar**
This creates a Contributor's Guide, Code of Conduct, and establishes the Developer Certificate of Origin.

### CODE REVIEW

**Comment by qedrakmar** [comment]
In keeping with the approval rules set forth in this merge, we'll require at least one approval before this gets merged.  Also, if there are any objections to the approval criteria, please voice them here.  Addendums should be made after this first pass is in place.

**Review by ztreinhart — APPROVED** [review]
This seems like a good starting point, @qedrakmar. Nice work!

**Comment by qedrakmar** [comment]
Barring other comments, I'll merge this end of day today.

**Pull request merged**
